client/src/pages/SignUp/SignUp.tsx

```diff
@@ -16,6 +17,18 @@ export default function Register(): JSX.Element {
   const { updateLoginContext } = useAuth();
   const { updateSnackBarMessage } = useSnackBar();
 
+  const handleDemoLogin = () => {
+    demoLogin('demo@kanban.com', '123456').then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        updateSnackBarMessage('Unexpected error! Please try again');
+      }
```

Comment:
This code is identical to the login one, we should be re-using this as opposed to duplicating this.